### PR TITLE
Added my event to calendar 

### PR DESCRIPTION
**Pull request by SahityaRoy**

### CODE REVIEW

File: public/data/SahityaRoy.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "type": "youtube",
+  "author": {
+    "name": "Sahitya Roy",
+    "url": "http://github.com/SahityaRoy"
+  },
+  "url": "http://twitter.com/SahityaRoy07",
```
Comment (eddiejaoude) [inline public/data/SahityaRoy.json:7]: This should go to the actual event itself not a general link

File: public/data/SahityaRoy.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  "title": "Amazing programming and technology releted vieos you can watch and learn ",
+  "start": "2021-11-06T17:00:00",
```
Comment (eddiejaoude) [inline public/data/SahityaRoy.json:9]: This event happened many weeks ago already? When is your next event?

**Review by suitupstan — CHANGES_REQUESTED** [review]
Please see the comments above 🙂

**Comment by eddiejaoude** [comment]
Event past, so I will close. Please feel free to create a new PR with up and coming event

**Pull request closed without merging**